admin_guide/out_of_resource_handling.adoc

```diff
@@ -220,6 +278,25 @@ If the latter, the node evicts a `Guaranteed` pod that least impacts node
 stability and limits the impact of the unexpected consumption to other
 `Guaranteed` pods.
 
+Local disk is a `BestEffort` resource.  If necessary, the node will evict
+pods one at a time to reclaim disk when `DiskPressure` is encountered.  The node
```

Comment:
this is currently not the case.  disk eviction currently doesn't work as evicted pods don't have their disk-based volumes cleaned up.  Remedied by https://github.com/kubernetes/kubernetes/pull/37228